backend/layers/processing/schema_migration.py

```diff
@@ -188,11 +199,14 @@ def publish_and_cleanup(self, collection_version_id: str, can_publish: bool) ->
                         "rollback": False,
                     }
                 )
-            elif dataset.status.processing_status != DatasetProcessingStatus.SUCCESS:
+            elif dataset.status.processing_status not in [
+                DatasetProcessingStatus.SUCCESS,
+                DatasetProcessingStatus.INITIALIZED,
+            ]:
```

Comment:
I think we can keep the old check now that we only look at datasets the migration processed; since we filter for datasets that were already successfully processed in collection_migrate, INITIALIZED would be a bad sign